.goreleaser.yml

```diff
@@ -154,8 +155,8 @@ release:
 
     ```
     cosign verify-blob \
-      -key https://raw.githubusercontent.com/smallstep/certificates/master/cosign.pub \
-      -signature ~/Downloads/step-ca_darwin_{{ .Version }}_amd64.tar.gz.sig
+      --certificate ~/Downloads/step-ca_darwin_{{ .Version }}_amd64.tar.gz.sig.pem \
+      --signature ~/Downloads/step-ca_darwin_{{ .Version }}_amd64.tar.gz.sig \
```

Comment:
Have you tried doing the `verify-blob` without the `--certificate` too? I think that should also work and should say something like 

```console
$ COSIGN_EXPERIMENTAL=1 cosign verify-blob --signature some-bin.sig some-bin
tlog entry verified with uuid: "2b7ca2bfb7ee09114a15d60761c2ade8c97f07cc20c02e635a92ba137a08a6de" index: 1234567
Verified OK
```

This depends on the Rekor server to be available (as does the signing part), so there's a potential failure there. We could add both methods for completeness?

Reply:
I just tried this locally and it didn't work.

```
 max@Maxs-MacBook-Pro  ~/src/github.com/smallstep/gitops   main  COSIGN_EXPERIMENTAL=1 cosign verify-blob --signature ~/Downloads/checksums.txt.sig ~/Downloads/checksums.txt
WARNING: No valid entries were found in rekor to verify this blob.

Transparency log support for blobs is experimental, and occasionally an entry isn't found even if one exists.

We recommend requesting the certificate/signature from the original signer of this blob and manually verifying with cosign verify-blob --cert [cert] --signature [signature].
Error: verifying blob [/Users/max/Downloads/checksums.txt]: could not find a valid tlog entry for provided blob, found 1 invalid entries
main.go:62: error during command execution: verifying blob [/Users/max/Downloads/checksums.txt]: could not find a valid tlog entry for provided blob, found 1 invalid entries

⬆ [ exit: 1 ] [ time: 10/26:16:18:18 ]
 max@Maxs-MacBook-Pro  ~/src/github.com/smallstep/gitops   main  COSIGN_EXPERIMENTAL=1 cosign verify-blob --signature ~/Downloads/checksums.txt.sig --certificate ~/Downloads/checksums.txt.pem  ~/Downloads/checksums.txt
tlog entry verified with uuid: bab7cf6963f44c45f617ec8dec222e8171dc741d95e46a66663ac9e0c6e9b7c5 index: 5881305
Verified OK
```

Reply:
FWIW this does work with verifying signatures on containers. `COSIGN_EXPERIMENTAL=1 cosign verify smallstep/step-sds:0.2.2-rc13` just works.

Reply:
I guess you hit one of the exceptional cases they mention in the error. Probably good to keep the `--certificate` as the main way to verify blobs now to not confuse people 🙂